### PR TITLE
[IMP] account-credit-control

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
security issue when using action.read()
replaced by for_xml_id

### CODE REVIEW

**Comment by xavier-bouquiaux** [comment]
@acsonefho 
@adrienpeiffer

File: account_credit_control/models/credit_control_line.py [hunk 1]
```diff
@@ -296,7 +296,7 @@ def button_credit_control_line_form(self):
         self.ensure_one()
         action = self.env.ref("account_credit_control.credit_control_line_action")
         form = self.env.ref("account_credit_control.credit_control_line_form")
-        action = action.read()[0]
+        action = self.env["ir.actions.act_window"]._for_xml_id(action)
```
Comment (adrienpeiffer) [inline account_credit_control/models/credit_control_line.py]: This one will raise an exception

File: account_credit_control/models/credit_control_run.py [hunk 2]
```diff
@@ -181,7 +181,7 @@ def open_credit_communications(self):
         action = self.env.ref(
             "account_credit_control.credit_control_communication_action"
         )
-        action = action.read()[0]
+        action = self.env["ir.actions.act_window"]._for_xml_id(action)
```
Comment (adrienpeiffer) [inline account_credit_control/models/credit_control_run.py]: Same here

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Review by adrienpeiffer — APPROVED** [review]

**Pull request merged**